sdk/avalon_sdk/direct/jrpc/jrpc_util.py

```diff
@@ -48,11 +26,12 @@ class JsonRpcErrorCode(Enum):
     UNSUPPORTED_MODE = 6
     KEY_EXISTS = 9
 
+def construct_message(status, message):
```

Comment:
This method name looks very generic for the functionality it implements. May be rename it to create_json_status_message ?

Reply:
Done